### ISSUE TITLE
Does not work on the mobile version of YouTube

### ISSUE THREAD

**Issue by sacbegg** [hallzy/remove-youtube-shorts#1]
Thanks for the code, it's the only script I found that will remove the shorts completely, but apparently it doesn't work with the mobile versions, is there any change in these versions?
I changed these lines in header to test and it still doesn't work. Thank you very much!

// @match *://*.youtube.com/*
// @exclude *://music.youtube.com/*
// @exclude *://*.music.youtube.com/*

**Comment by hallzy** [comment]
@sacbegg I never actually looked at the mobile site when I first made this, but I just updated the script and it should work on mobile now :)

**Comment by sacbegg** [comment]
Thanks!, although it doesn't seem to work, the shorts keep appearing :(

**Comment by hallzy** [comment]
Well, that's what I get for testing it on desktop with mobile emulation...

I just made an update and tried it on an actual iPhone this time with the Userscripts extension, and it looks like it is working for me now.

Turns out that the userscripts extension for iOS seems to not recognize scripts unless the file name ends with `.user.js`, so I just renamed the file and that seemed to work for me.

Hopefully that also works for you.
